src/libs/ck-libs/ampi/ampi.C

```diff
@@ -1364,6 +1368,50 @@ ampiParent::ampiParent(CkMigrateMessage *msg) noexcept
 #endif
 }
 
+#if CMK_AMPI_WITH_ROMIO
+void ADIO_Init_Globals(struct ADIO_GlobalStruct * globals)
+{
+  globals->ADIOI_Flatlist = NULL;
+  globals->ADIOI_Datarep_head = NULL;
+  /* list of datareps registered by the user */
+
+  /* for f2c and c2f conversion */
+  globals->ADIOI_Ftable = NULL;
+  globals->ADIOI_Ftable_ptr = 0;
+  globals->ADIOI_Ftable_max = 0;
+  globals->ADIOI_Reqtable = NULL;
+  globals->ADIOI_Reqtable_ptr = 0;
+  globals->ADIOI_Reqtable_max = 0;
+#ifndef HAVE_MPI_INFO
```

Comment:
Are these preprocessor symbols available when compiling AMPI itself?

Reply:
Do you mean `HAVE_MPI_INFO`? See the change to ampiimpl.h

Reply:
Maybe just add a comment (also ROMIO_XFS, ROMIO_LUSTRE ).